llama_index/response_synthesizers/google/generativeai/base.py

```diff
@@ -0,0 +1,94 @@
+import logging
+from typing import Any, List, Sequence, cast
+
+from llama_index.bridge.pydantic import BaseModel  # type: ignore
+from llama_index.prompts.mixin import PromptDictType
+from llama_index.response_synthesizers.base import BaseSynthesizer
+from llama_index.types import RESPONSE_TEXT_TYPE
+from llama_index.vector_stores.google.generativeai import google_service_context
+
+_logger = logging.getLogger(__name__)
+_import_err_msg = "`google.generativeai` package not found, please run `pip install google-generativeai`"
+_separator = "\n\n"
+
+
+class SynthesizedResponse(BaseModel):
+    answer: str
+    attributed_passages: List[str]
+    answerable_probability: float
+
+
+class GoogleTextSynthesizer(BaseSynthesizer):
```

Comment:
nit: same as the other file, we should add a class level file linking to https://developers.generativeai.google/api/python/google/generativeai/create_tuned_model